src/app/newsFeedPage/page.tsx

```diff
@@ -21,26 +22,67 @@ export default function App() {
     getNews();
   }, [news]);
 
+  const [windowWidth, setWindowWidth] = useState(window.innerWidth);
+  useEffect(() => {
+    const handleResize = () => setWindowWidth(window.innerWidth);
+    window.addEventListener('resize', handleResize);
+    return () => {
+      window.removeEventListener('resize', handleResize);
+    };
+  }, []);
+
   return (
-    <div className="bg-ivory h-screen">
+    <div className="bg-ivory">
```

Comment:
There's a split in background color.  At the bottom of the screen it is white instead of bg-ivory
You can better visualize this by setting bg-ivory to something high contrast like bg-red-500.

It looks like when you set h-auto, the size of this page is based on how many news displays are created, but there is a minimum screen height so the rest is padded out with a white screen.  Trying using h-full or h-screen so that your page always takes up the entire view height.
<img width="406" alt="image" src="https://github.com/calblueprint/phs/assets/73627326/211a6a5b-975a-43b9-8e31-2ecb17be9657">
<img width="411" alt="image" src="https://github.com/calblueprint/phs/assets/73627326/9425fcd0-28fd-4748-9cf5-615e8242b174">